build/fish/build.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2019 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/build.sh
+
+PROG=fish
+VER=4.0.9
+PKG=ooce/shell/fish
+SUMMARY="Fish is a smart and user-friendly command line shell"
+DESC="friendly interactive shell"
+
+SKIP_SSP_CHECK=1
+SKIP_LICENCES=COPYING
+
+BUILD_DEPENDS_IPS="ooce/developer/rust library/pcre2"
+
+set_arch 64
+
+CONFIGURE_OPTS+="
+	DFISH_USE_SYSTEM_PCRE2=ON
+"
+
+TESTSUITE_SED="
+    /^$/ {
+        N
+        /failures:/b op
+    }
+    d
+    :op
+    /^gmake:/d
+    /^Build completed/d
+    n
+    b op
+"
```

Comment:
you add a testsuite transform but there are no testsuite results in this commit?